src/arm/linux/cpuinfo.c

```diff
@@ -790,7 +790,7 @@ static bool parse_line(
 			} else if (memcmp(line_start, "Hardware", key_length) == 0) {
 				size_t value_length = value_end - value_start;
 				if (value_length > CPUINFO_HARDWARE_VALUE_MAX) {
-					cpuinfo_log_info(
+					cpuinfo_log_debug(
```

Comment:
Same as below, imo this should stay an info or even be elevated to warning.

Reply:
Splitting into separate PR as discussed above.